### PR TITLE
Refact temporary vars in ceph-common defaults.

### PR DESCRIPTION
**Pull request by guits**
These variables were defined here to be sure that
    `roles/ceph-common/tasks/checks/check_mandatory_vars.yml` has all variables defined.

### CODE REVIEW

**Comment by alfredodeza** [comment]
jenkins test this please

**Comment by alfredodeza** [comment]
jenkins test ansible2.2-dmcrypt_journal_collocation

**Comment by alfredodeza** [comment]
jenkins test ansible2.2-journal_collocation

**Comment by andrewschoen** [comment]
jenkins test this please

**Comment by andrewschoen** [comment]
jenkins test ansible2.2-xenial_cluster

File: roles/ceph-common/defaults/main.yml [hunk 1]
```diff
@@ -356,17 +356,6 @@ mon_containerized_default_ceph_conf_with_kv: false
 # should be removed after a refactor has properly seperated all the checks into
 # the appropriate roles.
 
-journal_collocation: False
```
Comment (alfredodeza) [inline roles/ceph-common/defaults/main.yml]: looks like the comment above can go as well?

**Pull request merged**